### DISCUSSION TITLE
Why is “crossing beam” position method so stable?

### DISCUSSION THREAD

**Discussion by nigeiwo** [bitcraze/discussions#27] [Q&A] [answered]
Hi, I have some questions about  “crossing beam” location method.  Why is this position method so stable?  What should be done when the positions of the four sensors on the deck are calculated, but one point is calculated far away from the other three points？

**Comment by whoenig** [comment, marked as answer]
The crossing beam method filters such lossy datapoints. The proper way is the EKF, which also keeps track of the uncertainty of the current pose estimation (this is important for outlier detection).

**Comment by jonasdn** [comment]
Thanks for answering @whoenig !

If you have more questions, feel free to repoen @nigeiwo !
